kibana-reports/package.json

```diff
@@ -1,49 +1,40 @@
 {
   "name": "opendistro_kibana_reports",
-  "version": "1.10.1.0",
+  "version": "1.12.0.0",
   "description": "Kibana Reporting Plugin",
   "license": "Apache-2.0",
-  "main": "index.js",
+  "main": "index.ts",
   "kibana": {
-    "version": "7.9.1",
+    "version": "7.10.0",
     "templateVersion": "1.0.0"
   },
   "scripts": {
-    "preinstall": "node ../../preinstall_check",
     "kbn": "node ../../scripts/kbn",
     "es": "node ../../scripts/es",
     "lint": "eslint .",
-    "start": "plugin-helpers start",
-    "test:server": "plugin-helpers test:server",
-    "test:browser": "plugin-helpers test:browser",
-    "build": "plugin-helpers build",
-    "test:jest": "../../node_modules/.bin/jest --config ./test/jest.config.js",
-    "test": "jest --config ./test/jest.config.js",
+    "start": "node ../../scripts/plugin_helpers start",
```

Comment:
You may add `plugin-helpers` script to avoid multiple  `node ../../scripts/plugin_helpers` below
```
"plugin-helpers": "node ../../scripts/plugin_helpers",

Reply:
good idea, thx!

Reply:
I look into some details, I can add the plugin_helper as script, but to avoid multiple `node ../../scripts/plugin_helpers`, what I need to do is actually variable reference in `package.json`. There seems like only one way to do it.
Ref: https://brianchildress.co/variables-in-package-json/

```
"config": {
    "plugin_helpers": "node ../../scripts/plugin_helpers"
  },
  "scripts": {
    ...
    "start": "$npm_package_config_plugin_helpers start",
    ...
}

```

Reply:
It looks to me that it's still very verbose, and the console output is less readable. You won't see the actual script getting executed, instead you see
![image](https://user-images.githubusercontent.com/32652829/99614078-9c2a8980-29cd-11eb-9de1-9c59f38dff33.png)

Reply:
https://github.com/joshuali925/trace-analytics/blob/7.10/package.json#L12-L17
you can try this, $env is not cross platform

Reply:
this is helpful, thx : )

Reply:
I‘m not sure if it works for you, I think what Yizhe said is like here https://github.com/elastic/kibana/tree/master/packages/kbn-plugin-helpers#installation